snorkel/classification/__init__.py

```diff
@@ -1,9 +1,18 @@
 """PyTorch-based multi-task learning framework for discriminative modeling."""
 
 from .data import DictDataLoader, DictDataset  # noqa: F401
+from .loss import (  # noqa: F401
+    cross_entropy_from_outputs,
```

Comment:
These are all user-facing?

Reply:
Yeah, the two standard losses (soft and hard) in _from_outputs format, and the raw loss function to be a general purpose corollary to F.cross_entropy().